examples/plot_pp_study.py

```diff
@@ -0,0 +1,371 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+================================
+Predictive Policing Case Study
+================================
+"""
+# %%
+#.. role:: raw-html-m2r(raw)
+#   :format: html
+#
+#Contents
+#========
+#
+#
+#1. Overview
+#2. Background
+#3. Fairlearn Results
+#4. Problem area & Sociotechnical context
+#5. Conclusion
+#
+#Overview
+#--------
+#
+#In this document we outline a sample sociotechnical case study - using a
+#hypothetical predictive policing scenario - based on data found in the
+#`Boston housing dataset <https://www.kaggle.com/c/boston-housing>`_
+#which we use in our scenario to predict per capita crime rate.
+#Discussion around the known fairness issues associated with this dataset
+#is included `here
+#<https://medium.com/@docintangible/racist-data-destruction-113e3eff54a8>`_.
+#For further questions about this document please contact Bruke Kifle
+#(@brkifle) or Michael Amoako (@miamoako). The full notebook containing
+#the code below is available upon request.
+#
+#Background
+#----------
+#
+#You are a contractor working with the Boston Police Department and
```

Comment:
I think that all these lines need a space after the `#` to start the comment.

Reply:
Current rendering: https://1951-133444044-gh.circle-artifacts.com/0/docs/_build/html/master/auto_examples/plot_pp_study.html#contents